readme.md

```diff
@@ -146,13 +172,13 @@ _Object path is property `a`, array index `0`, object property `b`._
 
 A React Native [`File`](https://developer.mozilla.org/docs/web/api/file) substitute for when using [`FormData`](https://developer.mozilla.org/docs/web/api/formdata).
 
-**Type:** [Object](https://mdn.io/object)
```

Comment:
@jaydenseric Any idea why the links got removed?

Reply:
Yes, the updated [`jsdoc-md`](https://www.npmjs.com/package/jsdoc-md) dev dependency no longer auto-links these types.